ssh-admin/Dockerfile

```diff
@@ -6,7 +6,7 @@ MAINTAINER Ilios Project Team <support@iliosproject.org>
 ENV GITHUB_ACCOUNT_SSH_USERS=''
 
 RUN apt-get update && \
-    apt-get install -y wget openssh-server sudo netcat mysql-client && \
+    apt-get install -y wget openssh-server sudo netcat mysql-client vim telnet && \
```

Comment:
Why telnet?

Actually - now that the edits are done is vim needed?

Reply:
they're just handy to have on the bastion server, for testing ports and editing stuff for quick tests...  I always miss them when i log in and want to "just test something really quick"